src/native/aws_signing.c

```diff
@@ -154,8 +154,11 @@ static void s_aws_complete_signing_result(
     struct aws_string *signature = NULL;
     aws_signing_result_get_property(result, g_aws_signature_property_name, &signature);
 
-    struct aws_byte_cursor signature_cursor = aws_byte_cursor_from_string(signature);
-    java_signature = aws_jni_byte_array_from_cursor(env, &signature_cursor);
+    /* Signature will be NULL for anonymous credentials. */
```

Comment:
Nit: for comments prefer saying why something is happening instead of static what is happening